maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -103,94 +93,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 			}
 
 			if (resolve) {
-				resolve(run);
+				run.resolve(failOnChanges);
```

Comment:
I am thinking we should have a bnd-resolve-maven-plugin which can be used to run the resolve and modify the source file.